FileUpload: Need to wrap CDR detect in if

Core is seeing 

```
rollbar.service.ts:76 Error: ViewDestroyedError: Attempt to use a destroyed view: detectChanges
    at viewDestroyedError (core.js:20354)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:23810)
    at checkAndUpdateView (core.js:23209)
    at callWithDebugContext (core.js:24079)
    at Object.debugCheckAndUpdateView [as checkAndUpdateView] (core.js:23781)
    at ViewRef_.push../node_modules/@angular/core/fesm5/core.js.ViewRef_.detectChanges (core.js:21590)
    at TsFileUploadComponent._this.updateInnerValue (terminus-ui-file-upload.js:658)
    at forms.js:3273
    at Array.forEach (<anonymous>)
    at FormControl.push../node_modules/@angular/forms/fesm5/forms.js.FormControl.setValue (forms.js:3273)
```

Which seems to be caused here:

https://github.com/GetTerminus/terminus-ui/blob/master/terminus-ui/file-upload/src/file-upload.component.ts#L491

We need to verify the CDR hasn't been destroyed. There are one or two examples in the UI library.

--- a/terminus-ui/file-upload/src/file-upload.component.ts
+++ b/terminus-ui/file-upload/src/file-upload.component.ts
@@ -488,7 +488,12 @@ export class TsFileUploadComponent extends TsReactiveFormBaseComponent implement
    */
   public updateInnerValue = (value: string): void => {
     this.value = value;
-    this.changeDetectorRef.detectChanges();
+
+    // NOTE: This `if` is to avoid: `Error: ViewDestroyedError: Attempt to use a destroyed view: detectChanges`
+    // istanbul ignore else
+    if (!this.changeDetectorRef['destroyed']) {
+      this.changeDetectorRef.detectChanges();
+    }
   }
 
 
@@ -501,7 +506,11 @@ export class TsFileUploadComponent extends TsReactiveFormBaseComponent implement
       this.formControl.valueChanges.pipe(
         untilComponentDestroyed(this),
       ).subscribe(() => {
-        this.changeDetectorRef.detectChanges();
+        // NOTE: This `if` is to avoid: `Error: ViewDestroyedError: Attempt to use a destroyed view: detectChanges`
+        // istanbul ignore else
+        if (!this.changeDetectorRef['destroyed']) {
+          this.changeDetectorRef.detectChanges();
+        }
       });
     }
   }

